hq/app/logic/VulnerableAccessKeys.scala

```diff
@@ -1,33 +1,42 @@
 package logic
 
 import config.Config.{iamHumanUserRotationCadence, iamMachineUserRotationCadence}
-import model.{AccessKey, AccessKeyEnabled, VulnerableAccessKey}
+import model.{AccessKey, AccessKeyEnabled}
 
 object VulnerableAccessKeys {
-
+  /**
+    * Flags active outdated keys for human users. Disabled keys are ignored.
+    */
   def hasOutdatedHumanKey(keys: List[AccessKey]): Boolean =
```

Comment:
Are we planning to remove `hasOutdatedHumanKey` and `hasOutdatedMachineKey`? I can't see where they're used.

Reply:
Yeah, if these aren't used then we can definitely get rid of them as well. I have a feeling they were still used when I did the original prune of unused code, but it looks like that's changed now 👍 

We can do another pass of tidying up straight after this, or as part of tidying up the unrecognised user job, whatever suits us best.